packages/front/src/routes/association/[identifier]/Association.controller.test.js

```diff
@@ -38,4 +43,28 @@ describe("Association Controller", () => {
             });
         });
     });
+
+    describe("getRnaSirenDuplicates()", () => {
+        const RNA_SIREN_2 = { rna: RNA_SIREN.rna, siren: "SIREN_2", id: "QNCQZ9822N49" };
+        const RNA_SIREN_3 = { rna: "RNA_2", siren: RNA_SIREN.siren, id: "MZD92Z0jj0àç" };
+        const mockGetAssociatedIdentifier = vi.fn(async () => [RNA_SIREN]);
```

Comment:
même idée qu'au dessus (et j'en vois d'autres je le mets pas partout mais le cœur y est)
```suggestion
        const mockGetAssociatedIdentifier = vi.fn(() => Promise.resolve([RNA_SIREN]));
```